CollAction/Controllers/ProjectsController.cs

```diff
@@ -415,7 +415,7 @@ public ViewResult StartInfo()
                     "Hi!<br><br>" +
                     "Thank you for participating in a CollAction project!<br><br>" +
                     "In crowdacting, we only act collectively when we meet the target before the deadline, so please feel very welcome to share this project on social media through the social media buttons on the project page!<br><br>" +
-                    "We�ll keep you updated on the project. Also feel free to Like us on <a href=\"https://www.facebook.com/collaction.org/\">Facebook</a> to stay up to date on everything CollAction!<br><br>" +
+                    "We�ll keep you updated on the project. Also feel free to Like us on <a href=\"https://www.facebook.com/collaction.org/\">Facebook</a> to stay up to date on everything CollAction!<br><br>" +
```

Comment:
My browser can't really view the character you replaced "-" and "’" with. Unicode error, just shows as a questionmark. Please undo?